app/assets/stylesheets/content/_in_place_editing.sass

```diff
@@ -265,6 +267,29 @@ a.inplace-editing--trigger-link,
     &.edit-strategy-textarea
       margin-top: auto
 
+// Explicit styles for comment / input-esque trigger appearance
+.work-packages--activity--add-comment
+  margin: 20px 0
+
+  .inplace-editing--container
+    border: 1px solid #eee
+
+  .inplace-edit--read-value
+    padding: 10px 5px
+
+  .inplace-editing--trigger-link
+    color: #888
+    font-size: 1rem
```

Comment:
This creates a deviation for comments:

![image](https://cloud.githubusercontent.com/assets/617519/9846423/57240ad4-5ad2-11e5-903d-088314a108a8.png)

from description

![image](https://cloud.githubusercontent.com/assets/617519/9846429/6279180c-5ad2-11e5-9304-060b0de88303.png)

and watchers

![image](https://cloud.githubusercontent.com/assets/617519/9846444/7b68bd72-5ad2-11e5-903e-af7c9aa10198.png)

Granted, description and watchers where different before but we shouldn't add a third version.